frontend/src/components/Login.js

```diff
@@ -1,31 +1,68 @@
 import React, { useState } from 'react'
 import { useAuth } from '../contexts/AuthContext'
 import { useNavigate } from 'react-router-dom'
+import Navbar from './Navbar'
+import {
+    Center,
+    Heading,
+    keyframes,
+    usePrefersReducedMotion,
+} from '@chakra-ui/react'
 
 const Login = () => {
     const [error, setError] = useState('')
     const [loading, setLoading] = useState(false)
     const { login } = useAuth()
     const navigate = useNavigate()
+    const prefersReducedMotion = usePrefersReducedMotion()
+
+    const bgKeyframe = keyframes`
+        0% {
+            background-position: 0% 50%;
+        }
+        50% {
+            background-position: 100% 50%;
+        }
+        100% {
+            background-position: 0% 50%;
+        }
+    `
 
     const handleLogin = async () => {
         setLoading(true)
         try {
             await login()
             navigate('/user')
         } catch {
-            setError('Failed to log in. Please retry.')
+            setError('Failed to log in, please retry')
         }
         setLoading(false)
     }
 
     return (
-        <div>
-            <button disabled={loading} onClick={handleLogin}>
-                Log In
-            </button>
-            {error && <span>{error}</span>}
-        </div>
+        <>
+            <Navbar
+                onClick={handleLogin}
```

Comment:
I cannot login ever, automatically fails

Reply:
This is caused by not having the Google OAuth sign-in method enabled in your Firebase Console. Readme has been updated to include instructions for how to set this up